.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
Ugh. The Java 7/8 compilation thing is going to be a pain inside google
until we have fully switched over to 8.  :/  not sure how we want to handle
that in our fork.  But we need to stay compiling on 7 until that magic
moment.

On Sun, Jan 11, 2015, 2:22 PM Thomas Broyer <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/square/javawriter/pull/160#discussion-diff-22770765>:
>
> > @@ -1,7 +1,6 @@
> >  language: java
> >
> >  jdk:
> > -  - oraclejdk7
>
> [image: :+1:]
>
> Too bad Travis CI doesn't support Maven Toolchains out of the box (
> travis-ci/travis-ci#2727
> <https://github.com/travis-ci/travis-ci/issues/2727>), we could have used
> a reactor build with 2 modules (or maybe even maven-invoker-plugin; note
> that this is possible by just shipping our own toolchains.xml).
> I think we can also achieve this with an explicit call to jdk_switcher
> use oraclejdk7 before running mvn test again in the script step of the
> .travis.yml.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/javawriter/pull/160/files#r22770765>.
>